build.sbt

```diff
@@ -98,7 +101,9 @@ lazy val flink111ManagerCompat = (project in file("flink111/manager")).
     dependencyOverrides ++= flinkOverrides(flink111V) ++ Seq(
       //For some strange reason, docker client libraries have conflict with schema registry client :/
       "org.glassfish.jersey.core" % "jersey-common" % "2.22.2",
-      "org.apache.kafka" % "kafka-clients" % "2.4.1"
+      "org.apache.kafka" % "kafka-clients" % "2.4.1",
+      // must be the same as used by flink - otherwise it is evicted by version from deployment-manager-api
```

Comment:
I wonder how will it work out with Nussknacker UI, where we already have newer akka?

Reply:
tested end-to-end, works like a charm